[Expo 35]Keep getting error Could not determine "artifacts for com.github.Raizlabs.DBFlow:dbflow:4.2.4"

My project is ejected from Expo 35.
I checked my ```app/build.gradle``` and I do not have any ```DBFlow```stuff in it.

But when I run ```gradlew build``` or ```gradlew test```. I am keeping getting this error:
```
Could not resolve all dependencies for configuration ':app:debugRuntimeClasspath'.
Could not determine artifacts for com.github.Raizlabs.DBFlow:dbflow:4.2.4
```
I am not familiar with gradle and I am not sure where this comes from. 
Can anyone help?

@NiciusB was that all you did? I've tried your way, deleting all the gradle caches I could find, and node_modules, and tried adding back with updated version as described above. Still, I always get this:

```AWS Device Farm configuration is NOT VALID

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':app:debugRuntimeClasspath'.
> Could not determine artifacts for com.github.Raizlabs.DBFlow:dbflow:4.2.4: Skipped due to earlier error
```
Not sure what the earlier error is supposed to be about.

I'm facing a same error.
In my case, my local build (`./gradlew bundleDebug` on my Mac) works but production build on Bitrise (`$ /Users/vagrant/git/android/gradlew ":app:bundleRelease"`) fails.

```
> Task :app:preReleaseBuild FAILED

FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:preReleaseBuild'.
> Could not resolve all dependencies for configuration ':app:releaseRuntimeClasspath'.
   > Could not determine artifacts for com.github.Raizlabs.DBFlow:dbflow:4.2.4: Skipped due to earlier error
```


I tried solutions above but did not work;
- Add dependenies of `com.github.Raizlabs.DBFlow` in app/build.gradle
- Clear cache before gradlew command by `expo publish --clear --release-channel my-production-channel`
- Update gradle version by opening the project with Android Studio


Does anyone know other things I can try?

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -68,6 +68,8 @@ allprojects {
     }
     // https://github.com/google/ExoPlayer/issues/5225#issuecomment-445739013
     maven { url 'https://google.bintray.com/exoplayer' }
+    // Using www.jitpack.io instead of plain jitpack.io due to
+    // https://github.com/jitpack/jitpack.io/issues/4002
     maven { url "https://www.jitpack.io" }
 
     // Want this last so that we never end up with a stale cache

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -68,7 +68,7 @@ allprojects {
     }
     // https://github.com/google/ExoPlayer/issues/5225#issuecomment-445739013
     maven { url 'https://google.bintray.com/exoplayer' }
-    maven { url "https://jitpack.io" }
+    maven { url "https://www.jitpack.io" }
 
     // Want this last so that we never end up with a stale cache
     mavenLocal()

--- a/android/expoview/build.gradle
+++ b/android/expoview/build.gradle
@@ -45,6 +45,11 @@ def safeExtGet(prop, fallback) {
 buildscript {
   repositories {
     maven { url 'https://maven.fabric.io/public' }
+    // If you have maven { url "https://jitpack.io" } as your resolving url,
+    // then Jitpack will only return the POM for the Android dependency causing the Gradle sync to fail.
+    // However, when you change url to "https://www.jitpack.io", Jitpack returns POM, AAR, and sources.jar.
+    // That is why we were adding www even though jitpacks docs don't say so.
+    // See https://github.com/jitpack/jitpack.io/issues/4002.
     maven { url "https://www.jitpack.io" }
     mavenCentral()
     google()

--- a/android/expoview/build.gradle
+++ b/android/expoview/build.gradle
@@ -45,7 +45,7 @@ def safeExtGet(prop, fallback) {
 buildscript {
   repositories {
     maven { url 'https://maven.fabric.io/public' }
-    maven { url "https://jitpack.io" }
+    maven { url "https://www.jitpack.io" }
     mavenCentral()
     google()
     jcenter()
@@ -57,7 +57,7 @@ buildscript {
 
 repositories {
   maven { url 'https://maven.fabric.io/public' }
-  maven { url "https://jitpack.io" }
+  maven { url "https://www.jitpack.io" }
   mavenCentral()
 }
 

--- a/android/versioned-abis/expoview-abi33_0_0/build.gradle
+++ b/android/versioned-abis/expoview-abi33_0_0/build.gradle
@@ -11,7 +11,7 @@ def safeExtGet(prop, fallback) {
 buildscript {
   repositories {
     maven { url 'https://maven.fabric.io/public' }
-    maven { url "https://jitpack.io" }
+    maven { url "https://www.jitpack.io" }
     mavenCentral()
     google()
     jcenter()
@@ -23,7 +23,7 @@ buildscript {
 
 repositories {
   maven { url 'https://maven.fabric.io/public' }
-  maven { url "https://jitpack.io" }
+  maven { url "https://www.jitpack.io" }
   mavenCentral()
 }
 

--- a/android/versioned-abis/expoview-abi34_0_0/build.gradle
+++ b/android/versioned-abis/expoview-abi34_0_0/build.gradle
@@ -11,7 +11,7 @@ def safeExtGet(prop, fallback) {
 buildscript {
   repositories {
     maven { url 'https://maven.fabric.io/public' }
-    maven { url "https://jitpack.io" }
+    maven { url "https://www.jitpack.io" }
     mavenCentral()
     google()
     jcenter()
@@ -23,7 +23,7 @@ buildscript {
 
 repositories {
   maven { url 'https://maven.fabric.io/public' }
-  maven { url "https://jitpack.io" }
+  maven { url "https://www.jitpack.io" }
   mavenCentral()
 }
 

--- a/android/versioned-abis/expoview-abi35_0_0/build.gradle
+++ b/android/versioned-abis/expoview-abi35_0_0/build.gradle
@@ -10,7 +10,7 @@ def safeExtGet(prop, fallback) {
 buildscript {
   repositories {
     maven { url 'https://maven.fabric.io/public' }
-    maven { url "https://jitpack.io" }
+    maven { url "https://www.jitpack.io" }
     mavenCentral()
     google()
     jcenter()
@@ -22,7 +22,7 @@ buildscript {
 
 repositories {
   maven { url 'https://maven.fabric.io/public' }
-  maven { url "https://jitpack.io" }
+  maven { url "https://www.jitpack.io" }
   mavenCentral()
 }
 

--- a/apps/expo-payments/android/build.gradle
+++ b/apps/expo-payments/android/build.gradle
@@ -67,7 +67,7 @@ allprojects {
     }
     // https://github.com/google/ExoPlayer/issues/5225#issuecomment-445739013
     maven { url 'https://google.bintray.com/exoplayer' }
-    maven { url "https://jitpack.io" }
+    maven { url "https://www.jitpack.io" }
 
     // Want this last so that we never end up with a stale cache
     mavenLocal()

--- a/docs/pages/versions/unversioned/sdk/payments.md
+++ b/docs/pages/versions/unversioned/sdk/payments.md
@@ -93,7 +93,7 @@ implementation project(':expo-payments-stripe')
     allprojects {
       repositories {
         ...
-        maven { url "https://jitpack.io" }
+        maven { url "https://www.jitpack.io" }
         ...
       }
     }

--- a/docs/pages/versions/v33.0.0/sdk/payments.md
+++ b/docs/pages/versions/v33.0.0/sdk/payments.md
@@ -96,7 +96,7 @@ implementation project(':expo-payments-stripe')
     allprojects {
       repositories {
         ...
-        maven { url "https://jitpack.io" }
+        maven { url "https://www.jitpack.io" }
         ...
       }
     }

--- a/docs/pages/versions/v34.0.0/sdk/payments.md
+++ b/docs/pages/versions/v34.0.0/sdk/payments.md
@@ -96,7 +96,7 @@ implementation project(':expo-payments-stripe')
     allprojects {
       repositories {
         ...
-        maven { url "https://jitpack.io" }
+        maven { url "https://www.jitpack.io" }
         ...
       }
     }

--- a/docs/pages/versions/v35.0.0/sdk/payments.md
+++ b/docs/pages/versions/v35.0.0/sdk/payments.md
@@ -96,7 +96,7 @@ implementation project(':expo-payments-stripe')
     allprojects {
       repositories {
         ...
-        maven { url "https://jitpack.io" }
+        maven { url "https://www.jitpack.io" }
         ...
       }
     }

